plugin/src/main/java/com/denizenscript/denizen/utilities/MultiVersionHelper1_20.java

```diff
@@ -0,0 +1,42 @@
+package com.denizenscript.denizen.utilities;
+
+import com.denizenscript.denizencore.objects.core.ElementTag;
+import com.denizenscript.denizencore.utilities.EnumHelper;
+import org.bukkit.Material;
+import org.bukkit.block.Sign;
+import com.denizenscript.denizencore.objects.core.ListTag;
+import com.denizenscript.denizencore.objects.core.MapTag;
+import org.bukkit.block.sign.Side;
+
+public class MultiVersionHelper1_20 {
+    public static boolean isSide(String side) {
+        return side != null && EnumHelper.get(Side.class).valuesMapLower.containsKey(EnumHelper.cleanKey(side));
+    }
+    public static ElementTag sideString(String side) {
+        return new ElementTag(Side.valueOf(side));
+    }

```

Comment:
This method seems redundant? the the output `ElementTag` will always have the same value as `String side` - why not just create a new `ElementTag` as usual?